mreuter/CSC_Testing/mt_utils.py

```diff
@@ -0,0 +1,122 @@
+from astropy.coordinates import AltAz, ICRS, EarthLocation, Angle
+from astropy.time import Time
+import astropy.units as u
+
+from lsst.ts import salobj
+
+def altaz_to_radec(alt, az):
+    location = EarthLocation.from_geodetic(
+        lon=-70.747698 * u.deg, lat=-30.244728 * u.deg, height=2663.0 * u.m
```

Comment:
These should really be in a constants.py file (like we've done for the AuxTel)...